manifest.jps

```diff
@@ -5,53 +5,58 @@ application:
   name: Shopozor k8s cluster
   version: 0.0
 
-  baseUrl: https://raw.githubusercontent.com/shopozor/services/dev
+  baseUrl: https://raw.githubusercontent.com/shopozor/services/setup-upfront-nginx
 
   settings:
     fields:
-      - name: envName
-        caption: Env Name
-        type: string
-        default: shopozor
-      - name: topo
-        type: radio-fieldset
-        values:
-          0-dev: '<b>Development:</b> one master (1) and one scalable worker (1+)'
-          1-prod: '<b>Production:</b> multi master (3) with API balancers (2+) and scalable workers (2+)'
-        default: 0-dev
-      - name: k8s-version
-        type: string
-        caption: k8s manifest version
-        default: v1.16.3
-      - name: cert-manager-version
-        type: string
-        caption: Cert-manager version
-        default: v0.13.1
-      - name: extDomains
-        type: string
-        caption: External domains (; separates)
-        required: true
+    - name: topo
+      type: radio-fieldset
+      values:
+        0-dev: '<b>Development:</b> one master (1) and one scalable worker (1+)'
+        1-prod: '<b>Production:</b> multi master (3) with API balancers (2+) and scalable workers (2+)'
+      default: 0-dev
+    - name: k8s-version
+      type: string
+      caption: k8s manifest version
+      default: v1.16.3
+    - name: extDomains
+      type: string
+      caption: External domains (; separates)
+      required: true
+    - name: nginxVersion
+      caption: Nginx version
+      type: string
+      required: true
+      default: 1.16.1
+    - name: openSslNumbits
+      caption: Numbits (OpenSSL)
+      type: numberpicker
+      required: true
+      default: 4096
+      editable: true
+      min: 0
+      max: 16384
+
+  globals:
+    DHPARAM_FILENAME: /var/lib/nginx/dhparam.pem
```

Comment:
This should not be necessary when we work with the cert-manager. Those parameters are provided by the jelastic kubernetes manifest.